src/pages/projects/common/hooks.tsx

```diff
@@ -246,7 +256,49 @@ export function useActions() {
     navigate('/projects/create?action=clone');
   };
 
+  const handleInvoiceProject = (project: Project) => {
+    toast.processing();
+
+    queryClient.fetchQuery({
+      queryKey: route('/api/v1/tasks?project_tasks=:projectId&per_page=100', {
+        projectId: project.id,
+      }),
+      queryFn: () =>
```

Comment:
Is there any reason we went with this instead of keeping consistency with rest of API calls?

Reply:
@beganovich Tbh, I research a bit about `fetchQuery` function and saw this convention on their web page. Now, I removed unnecessary parts and made convention same as the rest of the project. Changed!